src/main/java/com/answerdigital/answerking/exception/util/ErrorResponse.java

```diff
@@ -1,38 +1,31 @@
 package com.answerdigital.answerking.exception.util;
 
 import com.answerdigital.answerking.exception.AnswerKingException;
+import jakarta.servlet.http.HttpServletRequest;
 import lombok.Getter;
+import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ProblemDetail;
 
-import javax.servlet.http.HttpServletRequest;
+import java.net.URI;
 
 @Slf4j
 @Getter
-public class ErrorResponse {
-
-    private final String type;
-
-    private final String title;
-
-    private final int status;
-
-    private final String detail;
-
-    private final String instance;
-
+public class ErrorResponse extends ProblemDetail {
     private final String traceId;
 
+    @SneakyThrows
```

Comment:
Not huge, I was just wondering if it would be a little more beneficial to be more explicit with the SneakyThrows?

```suggestion
    @SneakyThrows(URISyntaxException.class)
```